index.md

```diff
@@ -39,4 +39,7 @@ redirect_from:
 - /engine/reference/commandline/cluster_rm/
 - /engine/reference/commandline/cluster_update/
 - /engine/reference/commandline/cluster_version/
+- /machine/
+- /kitematic/
+- /toolbox/
```

Comment:
This would also need all the `redirect_from` lines from the files we're removing added;

```suggestion
- /machine/
- /machine/overview/
- /machine/AVAILABLE_DRIVER_PLUGINS/
- /machine/DRIVER_SPEC/
- /machine/RELEASE/
- /machine/completion/
- /machine/concepts/
- /machine/drivers/
- /machine/drivers/aws/
- /machine/drivers/azure/
- /machine/drivers/digital-ocean/
- /machine/drivers/exoscale/
- /machine/drivers/gce/
- /machine/drivers/generic/
- /machine/drivers/hyper-v/
- /machine/drivers/index/
- /machine/drivers/linode/
- /machine/drivers/openstack/
- /machine/drivers/os-base/
- /machine/drivers/rackspace/
- /machine/drivers/soft-layer/
- /machine/drivers/virtualbox/
- /machine/drivers/vm-cloud/
- /machine/drivers/vm-fusion/
- /machine/drivers/vsphere/
- /machine/examples/aws/
- /machine/examples/index/
- /machine/examples/ocean/
- /machine/get-started-cloud/
- /machine/get-started/
- /machine/install-machine/
- /machine/migrate-to-machine/
- /machine/reference/active/
- /machine/reference/config/
- /machine/reference/create/
- /machine/reference/env/
- /machine/reference/help/
- /machine/reference/index/
- /machine/reference/inspect/
- /machine/reference/ip/
- /machine/reference/kill/
- /machine/reference/ls/
- /machine/reference/mount/
- /machine/reference/provision/
- /machine/reference/regenerate-certs/
- /machine/reference/restart/
- /machine/reference/rm/
- /machine/reference/scp/
- /machine/reference/ssh/
- /machine/reference/start/
- /machine/reference/status/
- /machine/reference/stop/
- /machine/reference/upgrade/
- /machine/reference/url/
- /machine/release-notes/
- /kitematic/
- /kitematic/faq/
- /kitematic/known-issues/
- /kitematic/minecraft-server/
- /kitematic/nginx-web-server/
- /kitematic/rethinkdb-dev-database/
- /kitematic/userguide/
- /toolbox/
- /docker-for-mac/docker-toolbox/
- /docker-for-windows/docker-toolbox/
- /mackit/docker-toolbox/
- /toolbox/faqs/
- /toolbox/faqs/troubleshoot/
- /toolbox/overview/
- /toolbox/toolbox_install_mac/
- /toolbox/toolbox_install_windows/
```

Reply:
> Alternatively;
> 
> * keep the kitematic/index.md, machine/index.md, and toolbox/index.md files but remove them from the TOC
> * and we can try if we can add a `redirect_to` to them that redirects to either the homepage or to the archived github repositories.
> 
> Let me give that one a quick try (as it would save us from having that huge list of redirects in the `/index.md`)

Looks like that works; opened as a PR here; https://github.com/docker/docker.github.io/pull/15112